.github/ISSUE_TEMPLATE/pipeline-issue-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Pipeline issue report
+about: Report a potential issue related to the pipeline editor or pipeline execution
+title: ''
+labels: component:pipeline-editor, component:pipeline-runtime, status:Needs Triage
```

Comment:
do we need these labels? Maybe add something relating this to user feedback/support?

Reply:
`kind:user feedback` adds another dimension, so that's probably a good one to add. I picked the two components to pre-classify the issue since the user indicated something unexpected happened related to them. Should, after triage, it turn out that the issue is not (for example) editor specific, we can just remove the tag to refine the classification.
I do agree that the component tags are not needed for general/unclassified issues, but they would not be opened using this template.